esp-hal/src/uart.rs

```diff
@@ -2628,6 +2649,17 @@ impl Info {
             .modify(|_, w| unsafe { w.stop_bit_num().bits(stop_bits as u8) });
     }
 
+    fn change_tx_idle(&self, idle_num: u16) {
+        // Bits 10:19 => 10-bit register has max value of 1023.
```

Comment:
- Is this true for all devices?
- Please see our API_GUIDELINES, and return an error instead of panicking.

Reply:
Still help to determine beyond than just `esp32`.

For error vs panic: [3205778](https://github.com/esp-rs/esp-hal/pull/2859/commits/32057789119978dbd8757e50f08d502673536d7c)